docker/px4-dev/Dockerfile_simulation

```diff
@@ -21,12 +21,16 @@ RUN wget --quiet http://packages.osrfoundation.org/gazebo.key -O - | apt-key add
 		python-numpy \
 	&& apt-get -y autoremove \
 	&& apt-get clean autoclean \
-	# px4tools, pymavlink
-	&& pip install px4tools pymavlink \
-	# dronekit
+	# px4tools
+	&& pip install px4tools \
```

Comment:
The next time this builds the docker image it will probably update px4tools to a newer version than what we were using, some api changes are included, so watch out for that. We might want to force a version with pip install px4tools==x.x.x or update the ci script for the new changes.